h2o-algos/src/main/java/hex/schemas/StackedEnsembleV99.java

```diff
@@ -45,6 +45,7 @@
       "metalearner_fold_column",
       "metalearner_params",
       "seed",
+      "training_scoring_subsample_size",
```

Comment:
In Deep Learning, we call this parameter, `score_training_samples` set to 10,000 by default (we can decide what a good default for SE is).  So we should use the same parameter name.

```
score_training_samples: Specify the number of training set samples for scoring. The value must be >= 0. To use all training samples, enter 0.
```

Reply:
ok, if there's a parameter with the same semantic for DL, then let's expose it, and use the same name.